languages/javascript/scripts/scaffold/template/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Is this the exact same we already use? If yes, you can just thumbs up this and resolve.